install/README.md

```diff
@@ -51,6 +51,10 @@ and set up an Istio IP range for outbound network access:
 * [Configure outbound network access](../serving/outbound-network-access.md)
 * [Configuring HTTPS with a custom certificate](../serving/using-an-ssl-cert.md)
 
+## Checking Your Installed Version of Knative
+
+* [Checking Your Installed Version of Knative](check-install-version.md)
```

Comment:
Since this topic only covers serving, maybe we can be clearer in this text ( -> "Knative Serving")